platform/stm32f0xx/ldscripts/default/stm32f0discovery_def.ld

```diff
@@ -6,14 +6,16 @@ Linker subscript for STM32F072 definitions with 128K Flash and 16K RAM
 
 ENTRY(Reset_Handler)
 
+/* TODO: explicitly set aside the last page for calib */
+
 MEMORY
 {
   FLASH (rx) : ORIGIN = 0x08000000, LENGTH = 128K
   RAM  (xrw) : ORIGIN = 0x20000000, LENGTH = 16K
 }
 
 /* highest address of the user mode stack */
-_estack = 0x20002000;
+_estack = 0x20004000;
```

Comment:
Where is this from? Can we just increase it like this?

Reply:
This was done on the `mockup` branch and I added it here to get it in master - we actually have 16KB of SRAM, addresses 0x20000000 to 0x20004000, but we previously only used half of it. It seemed to be a mistake left over from some MSXII (maybe even MSXI)-era migration from a smaller chip; they updated the rest of the linker script but didn't touch _estack for some reason.